src/interface/statistics/components/DonutChart/index.tsx

```diff
@@ -82,8 +98,8 @@ class DonutChart extends React.PureComponent {
           type: 'nominal',
           legend: null,
           scale: {
-            domain: items.map(({label}) => label),
-            range: items.map(({color}) => color),
+            domain: items.map(({ label }: any) => label),
```

Comment:
@Dambroda Not certain how to consolidate this with the rest of the typings, could you take a look if you have the time at some point :heart: